prime-router/src/main/kotlin/transport/SftpTransport.kt

```diff
@@ -306,6 +307,29 @@ class SftpTransport : ITransport, Logging {
         // allow us to mock SSHClient because there is no dependency injection in this class
         fun createDefaultSSHClient(): SSHClient {
             val sshConfig = DefaultConfig()
+
+            // Started from version 0.33.0, SSHJ doesn't try to determine RSA-SHA2-* support on fly.
+            // Instead, it looks only config.getKeyAlgorithms(), which may or may not contain ssh-rsa
+            // and rsa-sha2-* in any order.  The default config stops working with old servers like
+            // Apache SSHD that doesn't rsa-sha2-* signatures.  To make it works with old servers,
+            // we need to include the KeyAlgorithms.SSHRSA at the top of the list or have higher
+            // priority than other as below.
+            sshConfig.keyAlgorithms = listOf(
```

Comment:
Is it worth creating a tech debt ticket to make this configurable per receiver? I remember you saying this breaks after trying the first three key algorithms, is that still the case or is this not related to that?

Reply:
No, it isn't worth it to create a tech dept for this.  After this fix, it works with all of our reivers.